pkgs/servers/pulseaudio/qpaeq.nix

```diff
@@ -0,0 +1,45 @@
+{ mkDerivation
+, python3
+, fetchurl
+, lib
+, pulseaudio
+}:
+
+let
+  qp = "./src/utils/qpaeq";
+in
+mkDerivation rec {
+  pname = "qpaeq";
+  inherit (pulseaudio) version src;
+
+  buildInputs = [
+    (
+      (python3.withPackages(ps: with ps; [ pyqt5 dbus-python sip ]))
+      .override(args: { ignoreCollisions = true; })
+    )
+  ];
+
+  dontBuild = true;
+  dontConfigure = true;
+
+  installPhase = ''
+    install -D ${qp} $out/bin/qpaeq
+    '';
+
+  postPatch = ''
+    sed 's|dbus.mainloop.pyqt5|dbus.mainloop.glib|g' -i ${qp}
+    sed 's|DBusQtMainLoop|DBusGMainLoop|g' -i ${qp}
```

Comment:
Could convert both sed commands into a patch file?

Reply:
A patch file => something I need to maintain

A sed command => usually does it's job across even bigger changes

Reply:
What is the error message? I would expect the `dbus.mainloop.pyqt5` made available by `pyqt5` package.

Reply:
The issue with `sed` commands is though that they break silently while a patch file would show up, when somebody for instance try to update pulseaudio and the patch no longer applies.

Reply:
Works without the patch